adapters/cadent_aperture_mx/cadent_aperture_mx.go

```diff
@@ -1,4 +1,4 @@
-package emx_digital
+package cadent_aperture_mx
```

Comment:
From golang docs, https://go.dev/blog/package-names

> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps.

how about changing `cadent_aperture_mx` to `cadentaperturemx`

Reply:
Please note that in this scenario, the folder name and the bidder name must stay with the existing snake_case naming convention, as that is the bidder code we're using (`cadent_aperture_mx`) in Prebid.js.  As such, package name has changed, but directories are not renamed and yaml/json files are not renamed either.